__tests__/commands/config.js

```diff
@@ -49,3 +52,9 @@ test('set value "true" to an option', (): Promise<void> => {
     expect(config.registries.yarn.homeConfig['strict-ssl']).toBe(true);
   });
 });
+
+test('get prefix interprets tilde', (): Promise<void> => {
```

Comment:
i'm not sure what you're telling me here, i mean i'm not sure what to do with it D: